.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "root": true,
+  "extends": ["./node_modules/sanctuary-style/eslint-es3.json"],
+  "parserOptions": {"sourceType": "module"},
+  "env": {"node": true, "es6": true},
```

Comment:
Would it be interesting to use `"env": {"shared-node-browser": true, "es6": true}`?

Reply:
I did not know about this option. That's great. Let's add it to fluture-template and let it trickle down.